sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_unroll.cpp

```diff
@@ -8,7 +8,7 @@
 // UNSUPPORTED: target-nvidia, target-amd
 // UNSUPPORTED-INTENDED: aspect-ext_intel_matrix isn't currently supported for
 // other triples
-// XFAIL: run-mode && (gpu-intel-dg2 || arch-intel_gpu_bmg_g21)
+// XFAIL: run-mode && gpu && !arch-intel_gpu_bmg_g21
```

Comment:
Does it mean this test started to fail on PVC with the driver update?

Reply:
no it means it started to pass on bmg, i dont know what happens on pvc but it didnt change with the new driver